collective/ogtags/interfaces.py

```diff
@@ -0,0 +1,12 @@
+from zope.interface import Interface
+
+
+class IOGTagsImageProvider(Interface):
+    """Adapter to get ogtags image"""
+
+    def getOGTagsImage(self):
+        """ Get the Image used by ogtags.
+
+        @return: Image field
```

Comment:
Probably better to document the return value more explicitly. Something like: returns a tuple of (context, fieldname) or None.
And maybe mention something like this: Context is most likely to be the context that was passed in, but a custom implementation could return a parent folder or the first Image portal_type contained by the current context.
At least, I guess this is why you return the context as well, instead of only the fieldname.